bindata/egress-router/000-nad.yaml

```diff
@@ -13,11 +13,27 @@ spec:
     "name": "egress-router-cni-nad",
     "ip": {
       "addresses": [
-        "{{.Addresses}}"
+        "{{.Addresses}}/24"
         ],
       "destinations": [
-        "{{.AllowedDestinations}}"
+        {{ range $i, $val := .AllowedDestinations }}
```

Comment:
I would rather see this done in go rather than the templates. It's a lot easier to test.